### PR TITLE
ci: create issue on failing dependency tests

### PR DESCRIPTION
**Pull request by rflechtner**
## fixes https://github.com/KILTprotocol/ticket/issues/1826

This creates an issue on the sdk repository when the scheduled workflow run fails. Unfortunately I can't currently assign issues to our weekly board - the configuration allows it, but fails.

## How to test:
Look at run https://github.com/KILTprotocol/sdk-js/actions/runs/1947824073 and ticket https://github.com/KILTprotocol/sdk-js/issues/494

## Checklist:

- [x] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: .github/workflows/tests-polkadot-deps.yml [hunk 1]
```diff
@@ -77,10 +86,10 @@ jobs:
 
     strategy:
       matrix:
-        image: ['latest-rc']
-        required: ['optional']
+        image: ['latest']
+        required: ['required']
         include:
-          - image: 'latest-develop'
+          - image: 'latest-rc'
```
Comment (Diiaablo95) [inline .github/workflows/tests-polkadot-deps.yml:92]: `latest-rc` is now `latest-master`. I though I merged the change back into `develop`...

File: .github/workflows/tests-polkadot-deps.yml [hunk 2]
```diff
@@ -77,10 +86,10 @@ jobs:
 
     strategy:
       matrix:
-        image: ['latest-rc']
-        required: ['optional']
+        image: ['latest']
```
Comment (tjwelde) [inline .github/workflows/tests-polkadot-deps.yml:89]: latest-master required and latest-develop optional?

Comment (tjwelde) [reply .github/workflows/tests-polkadot-deps.yml:89]: waiting for answer regarding node tagging

**Review by tjwelde — APPROVED** [review]
Love it

File: .github/workflows/tests-polkadot-deps.yml (hunk 2 above)
Comment (tjwelde) [reply .github/workflows/tests-polkadot-deps.yml:89]: so, the `latest` tag moves forward with every release, so I think this is fine

Comment (rflechtner) [reply .github/workflows/tests-polkadot-deps.yml:89]: Ok I’ll merge then !

**Pull request merged**